src/components/schemas/expense.yaml

```diff
@@ -810,4 +810,155 @@ expense_creation_from_receipt_in:
+        This represents the cost_center's id given by Fyle during creation of cost center
+        for which this expense is created.
+      nullable: true
+      example: 1049
+    category_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_integer'
+      description: |
+        This represents the category's id given by Fyle during creation of category
+        for which this expense is created.
+      nullable: true
+      example: 49058
+    project_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_integer'
+      description: |
+        This represents the project's id given by Fyle during creation of project
+        for which this expense is created.
+      nullable: true
+      example: 92895
+    source_account_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_string'
+      description: |
+        This represents the source account's id given by Fyle during creation of account type
+        of user for which this expense is created.
+      example: acwbl222wlg
+    tax_amount:
+      type: number
+      nullable: true
+      description: Tax amount as setup by your organization.
+    tax_group_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_string'
+      description: |
+        This represents the Tax's group id given by Fyle during creation of tax group
+        for which this expense is created.
+      nullable: true
+    is_billable:
+      $ref: './fields.yaml#/is_billable'
+    is_reimbursable:
+      $ref: './fields.yaml#/is_reimbursable'
+    distance:
+      $ref: './fields.yaml#/distance'
+    distance_unit:
+      $ref: './fields.yaml#/distance_unit'
+    started_at:
+      allOf:
+        - $ref: './fields.yaml#/timestamptz_pst'
+      description: |
+        Specific to per diem related expenses, start date from which per diem claim is made.
+    ended_at:
+      allOf:
+        - $ref: './fields.yaml#/timestamptz_pst'
+      description: |
+        Specific to per diem related expenses, end date until which per diem claim is made.
+    travel_classes:
+      $ref: './fields.yaml#/travel_classes'
+    locations:
+      $ref: './fields.yaml#/locations'
+    custom_fields:
+      $ref: './fields.yaml#/custom_fields'
+    num_files:
+      type: integer
+      description: |
+        Number of files/reciept attached to expense
+      example: 4
+    is_matching_ccc:
+      type: boolean
+      description: |
+        This field is set to true if expense matches any card expense
+      default: false
+      example: false
+    mileage_rate_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_integer'
+      description: |
+        Specific to mileage expense. This represents the Mileage's rate id given by Fyle
+        during creation of mileage rate for which this expense is created.
+      nullable: true
+      example: 9080
+    mileage_calculated_distance:
+      type: number
+      nullable: true
+      description: Specific to mileage expense. Distance computed by Fyle in mileage expenses.
```

Comment:
Add TODO statement here, mentioning to update this content based on how do we decide to do mileage calculations when mileage support comes to platform world. For example:

- if we will give APIs for distance and amount calculation, then we should link those apis, in these description
- if we are going to auto compute it in backend, then these fields might need to be dropped
- etc. etc.

Reply:
same for below